lib/coxir/struct/channel.ex

```diff
@@ -160,6 +162,146 @@ defmodule Coxir.Struct.Channel do
     API.request(:post, "channels/#{channel}/messages/bulk-delete", %{messages: messages})
   end
 
+  @doc """
+  Modifies the name of a given channel.
+
+  Returns a channel object upon success
+  or a map containing error information.
+  """
+  @spec set_name(channel, String.t) :: map
+
+  def set_name(%{id: id}, name),
+    do: set_name(id, name)
+
+  def set_name(channel, name) do
```

Comment:
Can we make all the functions follow this format?
```elixir
def set_name(channel, name),
    do: edit(channel, name: name)
```

Reply:
Yes we can.